.gitignore

```diff
@@ -29,6 +29,7 @@
 .pub-cache/
 .pub/
 build/
+pubspec.lock
```

Comment:
I prefer to keep this file in git

Reply:
I removed it as it is recommended to not check the `pubspec.lock` file into VCS and I saw that you have also not checked it in in your other repo: https://github.com/letsar/flutter_sticky_header

If wanted, I can still commit it in again :)

Reply:
Yes you're right, it should not be in git except for applications. Let it in the gitignore then 🙂

Reply:
Perfect, thanks :)